server/src/mcr/feedback/feedback.server.spec.js

```diff
@@ -14,6 +14,9 @@ const PROPERTY = 'feedbacks'
 const BASE = '/api/feedback/'
 const ehrApp = new EhrApp()
 
+const visitId = Helper.sampleObjectId(true)
```

Comment:
Not used?  please remove

Reply:
Actually, it is being used, since I had previously added the auth middleware to the feedback end point. Therefore, to successfully pass the tests, it needs the authToken.